### PR TITLE
PATCH: use namespace from ray cluster

### PR DESCRIPTION
**Pull request by KPostOffice**

### CODE REVIEW

File: pkg/controllers/raycluster_webhook.go [hunk 1]
```diff
@@ -253,7 +253,7 @@ func oauthProxyContainer(rayCluster *rayv1.RayCluster) corev1.Container {
 			"--tls-cert=/etc/tls/private/tls.crt",
 			"--tls-key=/etc/tls/private/tls.key",
 			"--cookie-secret=$(COOKIE_SECRET)",
-			"--openshift-delegate-urls={\"/\":{\"resource\":\"pods\",\"namespace\":\"default\",\"verb\":\"get\"}}",
+			"--openshift-delegate-urls={\"/\":{\"resource\":\"pods\",\"namespace\":\"" + rayCluster.Namespace + "\",\"verb\":\"get\"}}",
```
Comment (franciscojavierarceo) [inline pkg/controllers/raycluster_webhook.go:256]: is this defaulted somewhere? I took a quick look and wasn't sure where it's configured. Not sure how that will affect the behavior but at least wanted to ask.

**Review by franciscojavierarceo — APPROVED** [review]

File: pkg/controllers/raycluster_webhook.go (hunk 1 above)
Comment (KPostOffice) [reply pkg/controllers/raycluster_webhook.go:256]: Is what defaulted?

Comment (franciscojavierarceo) [reply pkg/controllers/raycluster_webhook.go:256]: Sorry, let me clarify, where is the default value for `rayCluster.Namespace` set? I wanted to confirm it was set somewhere but I didn't immediately see it (I'm sure it is) but I figured I'd confirm with you to document it here on this PR.

Comment (KPostOffice) [reply pkg/controllers/raycluster_webhook.go:256]: Ahh, there is no default value for rayCluster.Namespace. It is a namespaced CR so it should always have a Name and a Namespace

Comment (franciscojavierarceo) [reply pkg/controllers/raycluster_webhook.go:256]: Cool, that makes sense! Just wanted to be sure. :)

**Pull request merged**